packages/components/src/Table/Cell/TableClickableCell.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * This component displays a cell of a table as a clickable element. The data must be a string.
+ * Callback methods (onClick, onDoubleClick and onKeyPress) must be passed via extra props.
+ */
+export default class TableClickableCell extends Component {
```

Comment:
This seems to be a default cell renderer.
For now you have to override the default renderer in order to have a click support. But what about a DefaultCellRenderer that manage the clicks if you have the callbacks in props. And no onClick will be passed to the div if they are undefined.
So that make it a merge of your 2 components TableCell and TableClickableCell.

Reply:
just to be sure, i merge these two components into only one TableCell component, that's right?

Reply:
Yes, to have only 1 component that deals with clicks if some props are provided